### PR TITLE
Populate README

### LINKED ISSUE #17: Populate README
The README should contain an explanation of what the repo is for and how to go about adding new tests/clients.

Waiting for #1.

### PR DESCRIPTION
**Pull request by tetsuo-cpp**
Closes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      - name: install
+        run: python -m pip install .
+      - uses: trailofbits/sigstore-conformance@v0.0.1
+        with:
+          entrypoint: sigstore
+```
+
+The only required configuration is the `entrypoint` parameter which provides a command to invoke
+the client. `sigstore-conformance` expects that the client exposes a CLI that conforms to the
+protocol outlined [here](docs/cli_protocol.md).
```
Comment (woodruffw) [inline README.md:30]: Making sure I'm not missing anything: this doc is intentionally missing from this PR, right?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+In the example above, the workflow is installing [sigstore-python](https://github.com/sigstore/sigstore-python)
+and providing `sigstore` as the `entrypoint` since this is the command used to invoke the client.
+
+Your workflow must have permission to request the OIDC token to authenticate with. This can be done
+by having a top-level `permission` setting for your workflow.
```
Comment (woodruffw) [inline README.md:36]: We can help the user restrict the token's scope here a bit, and tell them to only add it to the `conformance` job.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+`sigstore-conformance` is a conformance testing suite for Sigstore clients.
+
+## Usage
+
+Simply add `trailofbits/sigstore-conformance` to one of your workflows.
+
+```yaml
+jobs:
+  conformance:
```
[the remark this answers is not in the document]
Comment (woodruffw) [reply README.md:17]: Ref: [comment](https://github.com/trailofbits/sigstore-conformance/pull/26/files#r993493454)

File: README.md (hunk 1 above)
Comment (tetsuo-cpp) [reply README.md:30]: Yep, that's right. I'm going to add that as part of #11.

File: README.md (hunk 2 above)
Comment (tetsuo-cpp) [reply README.md]: [84b4013](https://github.com/trailofbits/sigstore-conformance/pull/26/commits/84b4013457d72f17b95a74efa9134c41454e60e5)

**Review by woodruffw — APPROVED** [review]
LGTM!

**Pull request merged**